src/pages/Proposals.jsx

```diff
@@ -28,6 +28,7 @@ const Proposals = React.memo(({
       as={RouterLink}
       to={`/dao/${daochain}/${daoid}/proposals/new`}
       rightIcon={<RiAddFill />}
+      title={customTerms ? 'proposal' : null}
```

Comment:
looks good. should these be capital case to feel like proper nouns?

Reply:
Oh yeah. Totally forgot about that .. Changing it now :)

Reply:
I think if we want to do it on buttons/links we should handle the whole button or link's text.

Reply:
I think if we want to do it on buttons/links we should handle the whole button or link's text.

let's get this to custom terms and then handle its translation.
<img width="185" alt="Screen Shot 2021-05-05 at 11 03 28 AM" src="https://user-images.githubusercontent.com/1778380/117173068-02477500-ad92-11eb-9be6-819e7cb22839.png">